src/CMakeLists.txt

```diff
@@ -337,14 +337,18 @@ if(WITH_TESTS)
     ../tests/util/CoreTest.cpp
     ../tests/util/DataTest.cpp
     ../tests/util/MathTest.cpp
+    ../tests/util/RandomTest.cpp
     ../tests/util/TimeTest.cpp
     ../tests/util/SchedulerTest.cpp
 	# Protocol/teleop tests
 	../tests/kinematics/DiffWristKinematicsTest.cpp)
 
   target_link_libraries(tests
     ${rover_libs}
-    stub_world_interface
+    stub_world_interface    double sample0 = stdn(0);
+    double sample1 = stdn(1);
+
+    REQUIRE(sample0 != sample1);
```

Comment:
This is the reason CMake isn't working. It looks like some code may have accidentally been written here. Remove it and it should work